pkgs/by-name/av/avdump3/package.nix

```diff
@@ -0,0 +1,43 @@
+{ lib, stdenv, writeScriptBin, fetchzip, dotnet-runtime, zlib }:
+
+stdenv.mkDerivation rec {
+  pname = "avdump3";
+  version = "8293_stable";
+
+  src = fetchzip {
+    url = "https://cdn.anidb.net/client/avdump3/avdump3_8293_stable.zip";
+    hash = "sha256-H9Sn3I4S9CmymKIMHVagDy+7svHs285S3EJgYQo+ks0=";
+    stripRoot = false;
+  };
+
+  installPhase = ''
+    mkdir -p $out/share/${pname} $out/bin
+    mv * $out/share/${pname}
+    cat > $out/bin/${pname} <<EOF
+    #!/bin/sh
+    export LD_LIBRARY_PATH="${lib.makeLibraryPath [ zlib ]}:\$LD_LIBRARY_PATH"
+    exec ${dotnet-runtime}/bin/dotnet $out/share/${pname}/AVDump3CL.dll "\$@"
+    EOF
+    chmod +x $out/bin/${pname}
+  '';
+
+  dontPatchELF = true;
+
+  meta = with lib; {
+    mainProgram = "avdump3";
+    description = "Tool for extracting audio/video metadata from media files and uploading it to AniDB";
+    longDescription = ''
+      AVDump is a tool to extract meta information from media files while at the
+      same time calculating multiple hashes. Based on that information reports
+      can be generated in multiple forms. Of particular interest is the ability
+      to send those reports back to AniDB and thereby quickly filling in missing
+      metadata for new files.
+    '';
+    homepage = "https://wiki.anidb.net/Avdump3";
+    sourceProvenance = with sourceTypes; [ binaryNativeCode binaryBytecode ];
+    # partial source code available under MIT license at https://github.com/DvdKhl/AVDump3
+    license = with licenses; [ mit unfree ];
+    maintainers = with maintainers; [ kini ];
+    platforms = [ "x86_64-linux" "aarch64-linux" ];
```

Comment:
Working for both platforms?

Reply:
`x86_64-linux` is working. I don't have an `aarch64-linux` machine to test, but I see there are `.so` files for it in the archive. Any suggestion what to do here? Just mark it as `x86_64-linux` exclusive for now and wait for some ARM user to come along and offer to test it on their platform?

Reply:
I asked for opinions in the NixOS matrix channel and the consensus seemed to be to restrict to `x86_64-linux` for now, so I updated the commit and pushed. Thanks for pointing it out!